### PR TITLE
WINC-1053: Add community 4.15 periodic with slack alerting

### PR DESCRIPTION
**Pull request by mansikulkarni96**

### CODE REVIEW

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.15__okd.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+base_images:
+  openshift_release_golang-1.19:
```
Comment (aravindhp) [inline ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.15__okd.yaml:2]: Shouldn't this be `openshift_release_golang-1.20` given that is the input below?

**Review by aravindhp — CHANGES_REQUESTED** [review]
Thanks for working on this, @mansikulkarni96

File: ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.15-periodics.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws
+    ci-operator.openshift.io/variant: okd
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-windows-machine-config-operator-community-4.15-okd-aws-e2e-operator
+  reporter_config:
+    slack:
+      channel: '#forum-winc'
```
Comment (alinaryan) [inline ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.15-periodics.yaml:19]: #forum-ocp-winc
And that reminds me that it should have been changed for the other branches too

**Review by alinaryan — COMMENTED** [review]
Thanks for working on this, 
Should we also make a 4.14 periodic?

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.15__okd.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  container:
+    from: src
+- as: build
+  commands: make build
+  container:
+    from: src
+- as: unit
+  commands: make unit
+  container:
+    from: src
```
Comment (aravindhp) [inline ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.15__okd.yaml:54]: I suspect we don't need these given there are no PRs opened and no presubmit tests. Not sure why they are present in other community operator configs.

**Review by aravindhp — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.15__okd.yaml (hunk 3 above)
Comment (alinaryan) [reply ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.15__okd.yaml:54]: We had them when we did community backports, but not needed anymore.

**Review by alinaryan — COMMENTED** [review]
LGTM. just a nit typo in commit message. 
podmman->podman

**Review by aravindhp — APPROVED** [review]

**Pull request merged**